cli/src/commands/auth/login.ts

```diff
@@ -38,7 +38,7 @@ export default class Login extends BaseCommand {
 
     await this.verifyAPIKey(key);
 
-    await writeAPIKey(key);
+    await setProfile({ apiKey: key });
```

Comment:
Auth login command hangs:

![image](https://user-images.githubusercontent.com/2181866/173629814-471051d0-7465-4aea-a6ad-8b7a1022f846.png)

Reply:
Yeah, it takes a couple of seconds to close existing http connections because the http server is gracefulled closed.

I've pushed this change https://github.com/xataio/client-ts/pull/301/commits/002b8408d2c9f25e52964b7c78efc83d7eafb632 and should be immediate now.